virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -7,28 +7,32 @@ toc::[]
 
 Use one of these procedures to create a virtual machine:
 
-* Running the virtual machine wizard
+* Quick Start guided tour
+* Running the wizard
 * Pasting a pre-configured YAML file with the virtual machine wizard
 * Using the CLI
 * xref:../../virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc#virt-importing-vmware-vm[Importing a VMware virtual machine or template with the virtual machine wizard]
 
 [WARNING]
 ====
 Do not create virtual machines in `openshift-*` namespaces.
-Instead, create a new namespace or use an existing namespace without the
-`openshift` prefix.
+Instead, create a new namespace or use an existing namespace without the `openshift` prefix.
 ====
 
+When you create virtual machines from the web console, select a virtual machine template that is configured with a boot source. Virtual machine templates with a boot source are labeled as *Available boot source* or they display a customized label text. Using templates with an available boot source expedites the process for creating virtual machines.
```

Comment:
s/process for creating/process of creating

Reply:
Change made.